src/Features/Core/Portable/Structure/BlockStructureOptions.cs

```diff
@@ -13,7 +13,8 @@ namespace Microsoft.CodeAnalysis.Structure
         bool ShowOutliningForCodeLevelConstructs = true,
         bool CollapseRegionsWhenFirstOpened = true,
         bool CollapseImportsWhenFirstOpened = false,
-        bool CollapseMetadataImplementationsWhenFirstOpened = true,
+        bool CollapseMetadataImplementationsWhenFirstOpened = false,
```

Comment:
❗ This should continue to be true. Otherwise, our move from Metadata As Source to Source Link / Decompiled Source essentially disables a standard feature that's been in place for ages.

Reply:
Looks like this is my only real remaining issue with this PR

```suggestion
        bool CollapseMetadataImplementationsWhenFirstOpened = true,
```